src/main/controllers/SubscriptionCaseSearchController.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express';
+import {SubscriptionCaseSearchActions} from '../resources/actions/subscriptionCaseSearchActions';
+import {isNull} from 'util';
+import {PipApi} from '../utils/PipApi';
+
+let _api: PipApi;
+
+export default class SubscriptionCaseSearchController {
+
+  constructor(private readonly api: PipApi) {
+    _api = this.api;
+  }
+
+  public get(req: Request, res: Response): void {
+    res.render('subscription-case-search');
+  }
+
+  public async post(req: Request, res: Response): Promise<void> {
+    const searchInput = req.body['search-input'];
+    if (searchInput && searchInput.length >= 3) {
```

Comment:
Just double checking, do we need a minimum length for the search on this one?

Reply:
There is no such a requirement, but we are checking min search length 3 on the other pages, so I followed the same pattern. It can be removed if necessary.